From 4c48f199052d97286b9b7a4cdc710be0666c4d3b Mon Sep 17 00:00:00 2001
From: Nikolay Nechaev <Nikolay_Nechaev@mail.ru>
Date: Sat, 4 Feb 2023 12:19:54 +0300
Subject: [PATCH 1/2] Moved notifications by a constant

A dummy patch to change the position of notifications at the bottom
---
 Telegram/SourceFiles/window/notifications_manager_default.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Telegram/SourceFiles/window/notifications_manager_default.cpp b/Telegram/SourceFiles/window/notifications_manager_default.cpp
index 2ce56b1..22f528d 100644
--- a/Telegram/SourceFiles/window/notifications_manager_default.cpp
+++ b/Telegram/SourceFiles/window/notifications_manager_default.cpp
@@ -59,7 +59,7 @@ namespace {
 	const auto x = (isLeft == rtl())
 		? (r.x() + r.width() - st::notifyWidth - st::notifyDeltaX)
 		: (r.x() + st::notifyDeltaX);
-	const auto y = isTop ? r.y() : (r.y() + r.height());
+	const auto y = isTop ? r.y() : (r.y() + r.height() - 20);  // Kolayne patch: - 20
 	return QPoint(x, y);
 }
 
-- 
2.44.0

